ci/example/README.md

```diff
@@ -0,0 +1,96 @@
+<img src="https://raw.githubusercontent.com/elbb/bb-buildingblock/master/.assets/logo.png" height="200">
+
+# Concourse pipeline integration guide
+
+For using this concourse CI popeline it is recommended to have two config files:
+
+-   `config.yaml`: for production pruposes
+-   `config.local.yaml`: for development and testing purposes
+
+The `config.yaml` is used to access the projects production release repository and branch and a real cloud S3 endpoint to store version artifacts.
+The `config.local.yaml` is used to develop and test the pipeline for building blocks. It is used with your private fork or local repository of the building block as well as a local S3 endpoint.
+
+# Example concourse CI pipeline
+
+This example is inteneded to make integration of the build pipeline more easy into other building blocks.
+
+## Adaptions for your building block
+
+This section describes what needs to be done within the several files. 
+**Note, that files calld `credentials.yaml` will is ignored by `.gitignore` and will not be checked in.**
+
+### config.example.yaml
+
+Open the `config.example.yaml` and specify the fields that are described.
+This config file shows an example for an fictive building block calld `bb-example`.
```

Comment:
"an fictive" -> "a fictive"
"calld" -> "called"

Reply:
"do you want do make it double?"
fixed.